src/app/shared/services/user/user.service.ts

```diff
@@ -18,6 +19,21 @@ export class UserService {
 
 
     logout() {
-        return this.http.post<void>(`/api/user/logout`, {});
+        return this.http.post<void>(`/api/user/logout`, {})
+            .pipe(tap(() => {
+                localStorage.removeItem('access_token');
+                localStorage.removeItem('refresh_token');
+            }));
+    }
+
+    refreshUserToken() {
+        return this.http.post<TokenModel>('/api/user/token/refresh', {
+            'refresh_token': localStorage.getItem('refresh_token')
+        })
+            .pipe(tap((data) => {
+                localStorage.setItem('access_token', data.access_token);
+                localStorage.setItem('refresh_token', data.refresh_token);
+                console.log('Refresh trigger!');
```

Comment:
Remove log.
In general, when you submit a pull request, try to pass along the changes you've made here, inside GitHub PR, so you can notice things like these. It happens a lot. Even if I have a new feature and have no reviewers to look at it, I would open a PR by myself and go over the changes. This way, I have one more step to check if everything looks fine.

Reply:
OK. Thnx for the tip!